server/routes/forms.ts

```diff
@@ -243,6 +243,21 @@ export default function routes({ componentNameService, serviceCatalogueService,
 }
 
 const buildFormData = (formData: Record<string, unknown>): GithubRepoRequestRequest => {
+  const githubProjectTeamsWrite = formData.github_project_teams_write
+    .toString()
```

Comment:
as this is done on 3 things, might be worth it being a function?

Reply:
replaced with function